src/core/lib/gpr/env_linux.cc

```diff
@@ -38,40 +38,29 @@
 #include "src/core/lib/gpr/string.h"
 #include "src/core/lib/gpr/useful.h"
 
-static const char* gpr_getenv_silent(const char* name, char** dst) {
```

Comment:
Sorry I don't fully know the history - but why to remove `gpr_getenv_silent` here? If I read this correctly, `gpr_getenv_silent` was a previous attempt to refactor some code that is no longer needed and is never called outside?

Reply:
Previously gRPC showed the warning message only when it's called from gpr_getenv but not from gpr_getenv_silent. Since this change removes it, it doesn't need to have two functions so _silent is removed.